Lesson 1/Homework/2.py

```diff
@@ -0,0 +1,5 @@
+if __name__ == '__main__':
+    print('python'*100)
+
```

Comment:
A lot of blank lines at the end of file. Should be only one blank line.

Reply:
done